### PR TITLE
Add More Thalmor Dossiers

### PR DESCRIPTION
**Pull request by loicreynier**
Add [More Thalmor Dossiers](https://www.nexusmods.com/skyrimspecialedition/mods/65059) entry and messages if provided patches are not installed.

### CODE REVIEW

File: masterlist.yaml [hunk 1]
```diff
@@ -15728,6 +15728,19 @@ plugins:
         util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 12
 
+  - name: 'More Thalmor Dossiers.esp'
+    url: [ 'https://www.nexusmods.com/skyrimspecialedition/mods/65059' ]
```
Comment (sibir-ine) [inline masterlist.yaml]: URLs should have trailing slashes for consistency & convenience if possible. It's also good to add location names now that they're utilized within the UI:

```suggestion
    url:
      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/65059/'
        name: 'More Thalmor Dossiers'
```

File: masterlist.yaml [hunk 2]
```diff
@@ -15728,6 +15728,19 @@ plugins:
...
+    clean:
+      - crc: 0xA057DD73
+        util: 'SSEEdit v4.0.4'
+    msg:
+      - <<: *patchProvided
+        subs: [ 'Helgen Reborn' ]
+        condition: 'active("Helgen Reborn.esp") and not active("More Thalmor Dossiers - Helgen Reborn.esp")'
+      - <<: *patchProvided
+        subs: [ 'Legacy of the Dragonborn' ]
+        condition: 'active("LegacyoftheDragonborn.esm") and not active("More Thalmor Dossiers - LotD.esp")'
```
Comment (sibir-ine) [inline masterlist.yaml]: The `clean` key should come after the `msg` key for consistency:

```suggestion
    msg:
      - <<: *patchProvided
        subs: [ 'Helgen Reborn' ]
        condition: 'active("Helgen Reborn.esp") and not active("More Thalmor Dossiers - Helgen Reborn.esp")'
      - <<: *patchProvided
        subs: [ 'Legacy of the Dragonborn' ]
        condition: 'active("LegacyoftheDragonborn.esm") and not active("More Thalmor Dossiers - LotD.esp")'
    clean:
      - crc: 0xA057DD73
        util: 'SSEEdit v4.0.4'
```

**Review by sibir-ine — CHANGES_REQUESTED** [review]

**Comment by loicreynier** [comment]
Thanks for the revisions, I'll keep that in mind.

**Review by sibir-ine — APPROVED** [review]
Thanks!

**Pull request merged**